cypress/support/pages/roles.ts

```diff
@@ -0,0 +1,47 @@
+import { keys } from "cypress/types/lodash";
+import { verify } from "cypress/types/sinon";
+import { randomNumber, role, smallRandomNumber, team, user } from "../../constants/globalConstants"
+
+//general data
+
+const name_1 = 'EA';
+const role_name = name_1+randomNumber;
+
+
+
+ //create a role
+Cypress.Commands.add('createARole', () => {
+  
+   //we are creating a super admin role
+
+   cy.get('#roles-link').click();
+   cy.get('#root > section > main > div > div > div > div > div.ant-table-title > header > div.cta-btns > a').click();
```

Comment:
very long paths like this are harder to read at a glance, and very prone to breaking. Usually, this is a sign that developers havent added specific enough classes/ids to crucial elements that youd want to access. If an element feels hard to access and that you have to do long paths like this, it's a very good idea to reach out to devs and ask them to add more specific identifiers